test/functionalTest/cases/0000_xml_parse/xmlParsePostNotifyContext.test

```diff
@@ -52,9 +52,9 @@ payload='<?xml version="1.0" encoding="UTF-8"?>
                 <value>1110</value>
               </contextMetadata>
               <contextMetadata>
-                <name></name>
-                <type></type>
-                <value></value>
+                <name>id2</name>
```

Comment:
These files that have changed (they had empty fields in metadata) are both input and output data for unit tests. The files used for input had to be changed, as with the new checks (the check method is the same for XML, JSONv1 and JSONv2) the unit test fails if the fields are empty.